protos/sigstore_common.proto

```diff
@@ -168,25 +168,17 @@ message SubjectAlternativeName {
         }
 }
 
-// A chain of X.509 certificates.
+// A collection of X.509 certificates.
+//
+// This "chain" can be used in multiple contexts, such as providing a root CA
+// certificate within a TUF root of trust or multiple untrusted certificates for
+// the purpose of chain building.
 message X509CertificateChain {
-        // The chain of certificates, with indices 0 to n.
-        // The first certificate in the array must be the leaf
-        // certificate used for signing.
-        //
-        // Signers MUST NOT include their root CA certificates in their embedded
-        // certificate chains, and SHOULD NOT include intermediate CA
-        // certificates that appear in independent roots of trust.
-        //
-        // Verifiers MUST validate the chain carefully to ensure that it chains
-        // up to a root CA certificate that they trust, regardless of whether
-        // the chain includes additional intermediate/root CA certificates.
-        // Verifiers MAY enforce additional constraints, such as requiring that
-        // all intermediate CA certificates appear in an independent root of
-        // trust.
+        // One or more DER-encoded certificates.
         //
-        // Verifiers SHOULD handle old or non-complying bundles that have
-        // additional intermediate/root CA certificates.
+        // In some contexts (such as `VerificationMaterial.certificate`), this sequence
```

Comment:
```suggestion
        // In some contexts (such as `VerificationMaterial.x509_certificate_chain`), this sequence
```

I believe this is what you meant? (Leaf is the first, then any intermediates.)

Reply:
Yes, thanks!